app/views/application/_object_info.html.erb

```diff
@@ -25,6 +25,17 @@
 	</li>
   <% end %>
 
+<% if current_object.can_have_content? %>
+  <li class="list-group-item list-group-item-#{original_filename_info[:context]}">
+	<span class="pull-right">
+		<% unless original_filename_info[:context] == 'info' %><em><% end %>
```

Comment:
The emphasis here seems inconsistent with the other object info styles -- or, if we do want that, can we do it with a CSS rule?